src/zenml/integrations/gcp/flavors/vertex_step_operator_flavor.py

```diff
@@ -28,28 +29,39 @@
     from zenml.integrations.gcp.step_operators import VertexStepOperator
 
 
-class VertexStepOperatorConfig(
+class VertexStepOperatorSettings(BaseSettings):
+    """Settings for the Vertex step operator.
+
+    Attributes:
+        accelerator_type: Accelerator type from list
+            https://cloud.google.com/vertex-ai/docs/reference/rest/v1/MachineSpec#AcceleratorType
+        accelerator_count: Defines number of accelerators to be used for the
+            job.
+        machine_type: Machine type specified here
+            https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types
```

Comment:
does it make sense to add this here or in docs [compatibility-table](https://cloud.google.com/vertex-ai/docs/training/configure-compute#gpu-compatibility-table)

Reply:
Absolutely, I'll add it!